samples/vxflex_v260_k8s_126.yaml

```diff
@@ -73,7 +73,15 @@ spec:
         # Default value: false
         - name: X_CSI_HEALTH_MONITOR_ENABLED
           value: "false"
-          
+
+        # X_CSI_APPROVE_SDC_ENABLED: Enables/Disable SDC approval
+        # Allowed values:
+        #    true: enable SDC approval
+        #    false: disable SDC approval
+        # Default value: false
+        - name: X_CSI_APPROVE_SDC_ENABLED
+          value: "false"
```

Comment:
Should this be within quotes? Allowed values are boolean

Reply:
yes , it should be in quotes